### PR TITLE
Add select component

### PR DESCRIPTION
**Pull request by chelliah**
## What is the Purpose?
Adds three versions of the select component, Basic, Multi-select, and Select with grouped options 

## Issue(s) affected?
#13

### CODE REVIEW

**Comment by chelliah** [comment]
Screen Shot 2020-04-15 at 5 28 01 PM
Screen Shot 2020-04-15 at 5 27 56 PM
Screen Shot 2020-04-15 at 5 27 52 PM
Screen Shot 2020-04-15 at 5 27 46 PM

File: src/components/select/BasicSelect.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  }
+};
+</script>
+
+<style scoped lang="scss">
+.el-select {
+  font-family: "Lato", sans-serif;
+  font-size: 16px;
+}
+</style>
```
Comment (andrewtpham) [inline src/components/select/BasicSelect.vue:48]: @chelliah can you try adding unbolding when an option is selected?

We're using `Lato light` so

```
.el-select-dropdown.is-multiple .el-select-dropdown__item.selected {
    font-weight: 300;
}
```

**Comment by andrewtpham** [comment]
@chelliah Also, once done, can you import these components and add them to `index.js`? This step is needed for when we publish to NPM. Thanks!

File: src/components/select/BasicSelect.vue (hunk 1 above)
Comment (andrewtpham) [reply src/components/select/BasicSelect.vue:48]: Resolving now and will add in my next PR

**Pull request merged**